libraries/ms-drivers/inc/drv120_relay.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+// Driver for the DRV120 relay
+// Requires GPIO and status to be initialized
+
+#include "gpio.h"
+#include "status.h"
+
+// Storage for GPIO pin to enable relay
+GpioAddress relay_pin;
```

Comment:
This should be declared in the `.c` file and should be static (and be prefixed with `s_`). It's the module's internal storage, so it doesn't need to be exported.